ui/mirage/factories/task.js

```diff
@@ -128,5 +130,9 @@ export default Factory.extend({
       });
       task.update({ schedule: schedule });
     }
+
+    if (task.withMeta) {
+      task.update({ meta: { raw: { foo: 'bar' } } });
+    }
```

Comment:
This is a factory file! Used by our mocking service (Mirage), and our tests, this gives us some private convenience methods when creating mocked objects like tasks/task groups. In this case, applying `withTaskMeta` to a task group will make a group with as many allocations/tasks as you'd like, each of which having some basic metadata.